project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ import sbt.*
 object Dependencies {
 
   val test: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc"          %% "ui-test-runner"        % "0.24.0",
+    "uk.gov.hmrc"          %% "ui-test-runner"        % "0.30.0",
```

Comment:
I believe 0.39.0 is the latest version

Reply:
ui-test-runner updated to latest version 0.38.0
https://catalogue.tax.service.gov.uk/repositories/ui-test-runner